interview-questions.md

```diff
@@ -7,61 +7,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer: An instance variable is run at the same time as the code is being typed. 
+  Your answer:
 
-  Researched answer: The main difference is its behavior with inheritance. Class variables are shared between a class and all its subclasses, while class instance variables only belong to tone specific class.
+  Researched answer:
 
 
 
 2. What is a block in Ruby?
 
-  Your answer: Ruby block is used in methods. (I honestly totally didn't know) 
+  Your answer:
 
-  Researched answer: Ruby block may appear only in the source adjacent to a method call. It is written on the same line as the method calls last parameter (the closing parentheses of the list). The code in the block is not executed at the same time it is encountered.
+  Researched answer:
 
```

Comment:
A Ruby block is code passed to a method. Think of the code that is passed to a higher-order method in JavaScript. 
```
JS         array.map(     < code >      )   ---- everything inside the ( ) is code that belongs to map
Ruby    array.map do    <code>    end  ----- everything inside the do/end is a block that is passed to map
```